### PR TITLE
Further UI improvements

### PR DESCRIPTION
**Pull request by TimGels**
This PR depends on #26.

### CODE REVIEW

File: Client/App.xaml.cs [hunk 1]
```diff
@@ -54,10 +55,15 @@ protected override void OnLaunched(LaunchActivatedEventArgs e)
...
-                    rootFrame.Navigate(typeof(MainPage), e.Arguments);
+                    //Go to mainpage when clienID isnt null. When it is not null it means that the user has a session.
+                    if (ApplicationData.Current.LocalSettings.Values["clientId"] != null)
+                    {
+                        rootFrame.Navigate(typeof(MainPage), e.Arguments);
+                    }
+                    else
+                    {
+                        rootFrame.Navigate(typeof(LoginPage), e.Arguments);
+                    }
```
Comment (SietzeKoonstra) [inline Client/App.xaml.cs:66]: We don't use sessions, it is also violating with the protocol. So this code isn't used. In my new branche the LoginPage is loaded by default as start page.

File: Client/Models/Client.cs [hunk 2]
```diff
@@ -37,17 +39,28 @@ public sealed class Client
...
+        {
+            get
+            {
+                return Id;
+            }
+            set
+            {
+                ApplicationData.Current.LocalSettings.Values["clientId"] = value;
+            }
+        }
```
Comment (SietzeKoonstra) [inline Client/Models/Client.cs:52]: ClientID is set when the client logged in successfully. We don't use sessions so this code isn't used.

File: Client/Services/Utilities/CustomItemContainerSelector.cs [hunk 3]
```diff
@@ -14,7 +14,7 @@ protected override Style SelectStyleCore(object item, DependencyObject container
             var message = (Message)item;
 
             //TODO: client ID halen uit client.
```
Comment (SietzeKoonstra) [inline Client/Services/Utilities/CustomItemContainerSelector.cs:16]: Todo can be removed.

File: Client/ViewModels/MainPageViewModel.cs [hunk 4]
```diff
@@ -75,49 +112,24 @@ public void CheckEnterPressed(object args)
...
+            Debug.WriteLine("OpenAddGroupView");
+            (Window.Current.Content as Frame).Navigate(typeof(LoginPage));
 
         }
 
+        private void OpenSignUpView()
+        {
+            Debug.WriteLine("OpenSignUpView");
+        }
+
```
Comment (SietzeKoonstra) [inline Client/ViewModels/MainPageViewModel.cs:124]: in new commits in ui-implementation-sietze this code has been updated.

**Review by SietzeKoonstra — APPROVED** [review]
lgtm for now.

**Comment by TimGels** [comment]
There are still some things to iron out. Changed the PR to a draft.

**Review by SietzeKoonstra — APPROVED** [review]
lgtm

**Review by SietzeKoonstra — APPROVED** [review]

**Review by JohannesKauffmann — APPROVED** [review]
Looks good.

**Review by SietzeKoonstra — COMMENTED** [review]
lgtm

**Pull request merged**